cypress/integration/organization_spec.js

```diff
@@ -91,9 +90,8 @@ describe('Organization Routers', () => {
 				compoundURL = Cypress.env('baseUrl').concat(
 					Cypress.env('version'),
 					Cypress.env('route_organizations'),
-					`?name=${createdOrgResponse.body.organization.name}&lastVerified=06/05/2021&createdAt=06/05/2021&lastUpdated=06/05/2021&pending=true&pendingOwnership=false&serviceArea=Medical,Legal&tags=Legal`
+					`?name=Surprise&lastVerified=06/05/2021&createdAt=06/05/2021&lastUpdated=06/05/2021&deleted=false&pending=true&pendingOwnership=false&serviceArea=Medical,Legal&tags=Legal`
```

Comment:
@Alfredo-Moreira should this be reverted to `${createdOrgResponse.body.organization.name}` ?